.github/scripts/test-coverage/coverage-report

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -euo pipefail
+
+committed_slices="$1"
+installed_slices="$2"
+output_file="${3:-coverage-report.json}"
```

Comment:
this script does need to use output files at all actually. it can just output to stdout and be redirected to a file if needed